source/tests/system/nidcpower_session_tests.cpp

```diff
@@ -9,8 +9,8 @@ namespace system {
 
 namespace dcpower = nidcpower_grpc;
 
-const int kTestViErrorRsrcNotFound = -1074134944;
-const char* kTestViErrorRsrcNotFoundMessage = "IVI: (Hex 0xBFFA0060) Insufficient location information or resource not present in the system.";
+const int kInvalidRsrc = -1074118656;
```

Comment:
Is there no error defined in the nidcpower.h or ivi.h that we could use?

Reply:
I couldn't find it in either of the header files. Codesearch didn't seem to return any helpful results for this error code either.